src/ViewModel/Button.php

```diff
@@ -124,6 +129,32 @@ public static function link(
         return $button;
     }
 
+    public static function action(
+        string $text,
```

Comment:
add path and id as parameters. id should be optional.

Reply:
Parameter path and id (optional) have been added.